lib/oli_web/live/users/user_detail_view.ex

```diff
@@ -82,35 +77,23 @@ defmodule OliWeb.Users.UsersDetailView do
             <ReadOnly label="Last Name" value={@user.family_name}/>
             <ReadOnly label="Email" value={@user.email}/>
             <ReadOnly label="Guest" value={boolean(@user.guest)}/>
-            <div class="form-control mb-2">
+            <div class="form-control mb-3">
               <Field name={:independent_learner}>
                 <Checkbox/>
                 <Label class="form-check-label mr-2">Independent Learner</Label>
               </Field>
             </div>
 
-            <section>
+            <section class="mb-2">
               <heading>
-                <p>Enable LMS-Lite Section Creation <small>(check both)</small></p>
+                <p>Enable Independent Section Creation</p>
+                <small>This ability allows delivery users to create and deliver "Independent" sections through OLI without a connected LMS, either Open and Free or limited to students who enroll through an invitation link.</small>
```

Comment:
I think we need to remove the reference to OLI here. Maybe something like:

"Allow this user to create "Independent" sections and enroll students via invitation link without an LMS"